src/overlays/actors/ovl_Bg_Crace_Movebg/z_bg_crace_movebg.h

```diff
@@ -2,19 +2,30 @@
 #define Z_BG_CRACE_MOVEBG_H
 
 #include "global.h"
+#include "assets/objects/object_crace_object/object_crace_object.h"
 
 struct BgCraceMovebg;
 
 typedef void (*BgCraceMovebgActionFunc)(struct BgCraceMovebg*, PlayState*);
 
+#define BGCRACEMOVEBG_GET_F(thisx) (((thisx)->params) & 0xF)
+#define BGCRACEMOVEBG_GET_7F0(thisx) (((thisx)->params >> 4) & 0x7F)
+#define BGCRACEMOVEBG_GET_3F80(thisx) (((thisx)->params >> 7) & 0x7F)
+#define BGCRACEMOVEBG_GET_F800(thisx) (((thisx)->params >> 0xB) & 0x1F)
+
 typedef struct BgCraceMovebg {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x18];
+    /* 0x000 */ DynaPolyActor dyna;
     /* 0x15C */ BgCraceMovebgActionFunc actionFunc;
-    /* 0x160 */ char unk_160[0x10];
-    /* 0x170 */ s32 unk_170;
-    /* 0x174 */ char unk_174[0x20];
-} BgCraceMovebg; // size = 0x194
+    /* 0x160 */ f32 unk160;
+    /* 0x164 */ f32 unk164;
+    /* 0x168 */ UNK_TYPE1 pad168[0x4];
+    /* 0x16C */ f32 unk16C;
+    /* 0x170 */ s32 unk170;
+    /* 0x174 */ s32 unk174;
+    /* 0x178 */ Vec3f unk178;
+    /* 0x184 */ u8 unk184;
+    /* 0x188 */ Vec3f unk188;
+} BgCraceMovebg; /* size = 0x194 */
```

Comment:
```suggestion
} BgCraceMovebg; // size = 0x194
```
Not sure how this got changed, but this is the style of `size =` you'll want